### PR TITLE
edit enigma problem statement

### PR DESCRIPTION
**Pull request by ghost**
modified:   index.html for Enigma

### CODE REVIEW

**Comment by ghost** [comment]
Edited index.html for Enigma. @shreyasbapat please have a look at this.

**Comment by shreyasbapat** [comment]
Right. This will need a rebase!

File: events/enigma/index.html [hunk 1]
```diff
@@ -1,305 +1,263 @@
...
+    <div class="row">
+        <div class="col-12">
+            <div class="upcoming-events">
+                <div class="upcoming-events-header">
+                    <h4>Problem Statement</h4>
+                </div>
+                <div class="upcoming-events-list ps">
+                    <div class="upcoming-event-wrap flex flex-wrap justify-content-between align-items-center">
+                        <header class="entry-header">
+                            <h4 class="entry-title">You will be given a dataset and 3-5 tasks on it (Exact number of problems will be released on site).
```
Comment (shreyasbapat) [inline events/enigma/index.html]: Don't mention giving a dataset.

File: events/enigma/index.html [hunk 2]
```diff
@@ -1,305 +1,263 @@
...
+                                 The solution of each problem will be evaluated and considered. The weights of each task shall be revealed at the competetion
+                                  site itslef. The participants are expected to develop an interface for easy input and output of the data. Top 8 teams shall
+                                   qualify for the second round involving a 4 minute presenttion and a short QnA. the event requires zero pre-event preparation
```
Comment (shreyasbapat) [inline events/enigma/index.html]: The spelling of presentation!

File: events/enigma/index.html [hunk 3]
```diff
@@ -1,305 +1,263 @@
...
+                    <div class="upcoming-event-wrap flex flex-wrap justify-content-between align-items-center">
+                        <header class="entry-header">
+                            <h4 class="entry-title">
+                                <ul class="ps-list">
+                                    <li>Participants must be present at the specified venue 10 minutes before the commencement of the event. Point distribution of tasks will be revealed.</li>                           
+                                    <li>15 minutes for the introduction of the problem statement and Q&A.</li>
+                                    <li>The duration of Enigma will be 5 hours.</li>
+                                    <li>Top teams will be given 10 minutes in the project presentation round.</li>
+                                    <li>Note: The times are subject to change depending on the number of participants.</li>
+                                </ul>
```
Comment (shreyasbapat) [inline events/enigma/index.html:144]: Add something like, please ensure you have the development environments set up beforehand. 5 hours is less, you don't get time to do that!

**Review by shreyasbapat — CHANGES_REQUESTED** [review]
I added some comments.

cc @lalitnarayanmudgal @akshat157

**Pull request closed without merging**